cmd/commands/subvolume.go

```diff
@@ -40,11 +40,10 @@ var listCmd = &cobra.Command{
 }
 
 var deleteCmd = &cobra.Command{
-	Use:                "delete",
-	Short:              "Deletes a stale subvolume.",
-	DisableFlagParsing: true,
-	Args:               cobra.ExactArgs(3),
-	Example:            "kubectl rook-ceph delete <subvolumes> <filesystem> <subvolumegroup>",
```

Comment:
Was deleting multiple subvolumes working before? Is that still a goal?

Reply:
yes it was working before, but as @Madhu-1 mentioned, it's better to follow single deletion at a time.

Reply:
The only way it really would make sense to support deletion of multiple subvolumes would be if the subvolumes were the last param, but I agree single deletion is sufficient.